tutorials/in_depth/tutorial08_stdp.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "*Copyright (C) 2021 Intel Corporation*<br>\n",
+    "*SPDX-License-Identifier: BSD-3-Clause*<br>\n",
+    "*See: https://spdx.org/licenses/*\n",
+    "\n",
+    "---\n",
+    "\n",
+    "# Online learning in Lava\n",
+    "\n",
+    "_**Motivation**: In this tutorial, we will demonstrate usage of a software model of Loihi's learning engine, exposed in Lava. This involves the LearningRule object for learning rule and other learning-related information encapsulation and the LearningConn Lava Process modelling learning-enabled connections._\n",
+    "\n",
+    "#### This tutorial assumes that you:\n",
+    "- have the [Lava framework installed](../../in_depth/tutorial01_installing_lava.ipynb \"Tutorial on Installing Lava\")\n",
+    "- are familiar with the [Process concept in Lava](../../in_depth/tutorial02_processes.ipynb \"Tutorial on Processes\")\n",
+    "- are familiar with the [ProcessModel concept in Lava](../../in_depth/tutorial02_process_models.ipynb \"Tutorial on ProcessModels\")\n",
+    "- are familiar with how to [connect Lava Processes](../../in_depth/tutorial05_connect_processes.ipynb \"Tutorial on connecting Processes\")\n",
+    "\n",
+    "#### This tutorial gives a bird's-eye view of\n",
+    "- Make use of the available learning rules in Lavas Process Library\n",
+    "- Develop custom learning rules for Loihi\n",
+    "\n",
+    "For this purpose, we will create a network of LIF and Dense processes with one plastic connection and generate frozen patterns of activity. We can easily choose between a floating point simulation of the learning engine and a fixed point simulation, which approximates the behavior on the Loihi1 neuromorphic hardware. We also will create monitors to observe the behavior of the weights and activity traces of the neurons and learning rules."
```

Comment:
Aren't we approximating the behavior of Loihi in general and not just Loihi1? May be we can change "Loihi1" to "Loihi"

Reply:
done